React/ReactRedux/Should I keep all component's state in Redux store.md

```diff
@@ -0,0 +1,7 @@
+## Should I keep all component's state in Redux store?
+
+You should:
```

Comment:
You haven't answered on the question, should you keep all components state in Redux or not?